### ISSUE TITLE
Xcode 9 beta 3 errors

### ISSUE THREAD

**Issue by huguesbr** [Polidea/RxBluetoothKit#123]
RxBluetoothKit was working great with `Xcode 9 beta 2` but after updating to `beta 3`, I got the following errors:

```
.../RxBluetoothKit/Source/Observable+QueueSubscribeOn.swift:78:7: Type 'QueueSubscribeOn<Element>' does not conform to protocol 'ObservableType'
```

```
.../RxBluetoothKit/Source/Observable+QueueSubscribeOn.swift:132:70: Cannot convert value of type 'QueueSubscribeOn<Element>' to expected argument type '(Event<_>) -> Void'
```

```
.../RxBluetoothKit/Source/Observable+QueueSubscribeOn.swift:164:76: Cannot convert return expression of type 'Observable<QueueSubscribeOn<Self.E>.E>' to return type 'Observable<Self.E>'
```

Any ideas?

**Comment by Kacper20** [comment]
Hi @huguesbr 

We have already PR for this - I will merge it in two hours and release next version.

Thanks for the patience,

Kacper

**Comment by Kacper20** [comment]
Hi,

It's already released as `3.0.14` version. 

Thanks again!

If you feel your issue has been resolved - please close the issue :)

**Comment by huguesbr** [comment]
yep problem solved, thx! 👍
